src/javascript/crypto/e2e/extension/ui/glass/glasswrapper.js

```diff
@@ -93,24 +86,20 @@ ui.GlassWrapper.prototype.installGlass = function(opt_callback) {
   glassFrame.scrolling = 'no';
   goog.style.setSize(glassFrame, goog.style.getSize(this.targetElem_));
   glassFrame.style.border = 0;
-  var pgpMessage = '';
-  var surroundings = ['', ''];
-
-  if (this.originalText_) {
-    pgpMessage = this.originalText_;
-  } else {
-    pgpMessage =
-        e2e.openpgp.asciiArmor.extractPgpBlock(this.targetElem_.innerText);
-    surroundings = this.targetElem_.innerText.split(pgpMessage);
-  }
+
+  var pgpMessage =
+      e2e.openpgp.asciiArmor.extractPgpBlock(this.targetElem_.innerText);
+  var surroundings = this.targetElem_.innerText.split(pgpMessage);
```

Comment:
after debugging, the problem found at `extractPgpBlock()` and `String.split()`

extractPgpBlock applies right trim to the pgp block and returns `pgpMessage`
https://github.com/yahoo/end-to-end/blob/master/src/javascript/crypto/e2e/openpgp/asciiarmor.js#L363

so `pgpMessage` can never be successful in splitting `this.targetElem_.innerText`, and as a result, the surronding texts are still missing.